### ISSUE TITLE
Composer Install throws Deprecation notice

### ISSUE THREAD

**Issue by bretterer** [sculpin/sculpin-blog-skeleton#49]
After doing a clone of the repo, and doing a `composer install` it returns
```
Deprecation Notice: The callback Sculpin\Composer\SculpinThemePlugin\SculpinThemePlugin::onPostPackageInstall declared at /Users/bretterer/Code/brianretterer-com/vendor/sculpin/sculpin-theme-composer-plugin/src/Sculpin/Composer/SculpinThemePlugin/SculpinThemePlugin.php accepts a Composer\Script\PackageEvent but post-package-install events use a Composer\Installer\PackageEvent instance. Please adjust your type hint accordingly, see https://getcomposer.org/doc/articles/scripts.md#event-classes in phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:322
Deprecation Notice: The callback Sculpin\Composer\SculpinThemePlugin\SculpinThemePlugin::onPrePackageInstall declared at /Users/bretterer/Code/brianretterer-com/vendor/sculpin/sculpin-theme-composer-plugin/src/Sculpin/Composer/SculpinThemePlugin/SculpinThemePlugin.php accepts a Composer\Script\PackageEvent but pre-package-install events use a Composer\Installer\PackageEvent instance. Please adjust your type hint accordingly, see https://getcomposer.org/doc/articles/scripts.md#event-classes in phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:322
```

**Comment by peter279k** [comment]
I also have the same issue there.
Perhaps it can ignore that because it's just a notice?

**Comment by beryllium** [comment]
This should be fixed by #52 (big thanks to @railto!). The minimum PHP version for SculpinBlogSkeleton is now 5.6+ as a result of this.
